src/iop/atrous.c

```diff
@@ -220,29 +209,27 @@ static inline __m128 weight_sse2(const __m128 *c1, const __m128 *c2, const float
 #if defined(__SSE2__)
 #define SUM_PIXEL_CONTRIBUTION_COMMON_SSE2(ii, jj)                                                           \
   {                                                                                                          \
-    const __m128 f = _mm_set1_ps(filter[(ii)] * filter[(jj)]);                                               \
-    const __m128 wp = weight_sse2(px, px2, sharpen);                                                         \
-    const __m128 w = _mm_mul_ps(f, wp);                                                                      \
-    const __m128 pd = _mm_mul_ps(w, *px2);                                                                   \
-    sum = _mm_add_ps(sum, pd);                                                                               \
-    wgt = _mm_add_ps(wgt, w);                                                                                \
+    const float f = filter[(ii)] * filter[(jj)];                                                             \
```

Comment:
const __m128 f = _mm_set1_ps(filter[(ii)] * filter[(jj)]);

Reply:
thanks ! That's odd though, because I use the same `float` scalar × `__m128` vector in colorbalance and filmic, and it never complained

Reply:
Hum... Where? I can't see one... Although I could have missed one...

Reply:
seems like you found them :-) done in last commit